Add docs for "List modules" (#39)
gacela-project/gacela-project.com: content/docs/gacela-script.md

+++
title = "Gacela script"

> description = "Gacela brings out of the box a code generator utility"

Reviewer: The description is odd 😕

Author: Ups, I forgot to update that 🥲

> description = "Gacela brings out of the box a code generator utility"

Reviewer: ```suggestion
description = "Out-of-the-box utility scripts that Gacela brings"
```
